pages/common/ssh-agent.md

```diff
@@ -0,0 +1,18 @@
+# ssh-agent
+
+> Spawn an ssh-agent process.
+> The ssh-agent will hold your ssh key decrypted in memory until evicted.
+> Add and manage keys with `ssh-add`.
```

Comment:
Perhaps this will read better?
```suggestion
> Spawn an SSH Agent process.
> An SSH Agent holds ssh keys decrypted in memory until removed or the process is killed.
> See also `ssh-add`, which can add and manage keys held by an SSH Agent.
```